boot/bootdata/hivecls.inf

```diff
@@ -597,6 +597,22 @@ HKCR,"CLSID\{A65B8071-3BFE-4213-9A5B-491DA4461CA7}\InProcServer32","ThreadingMod
 HKCR,"CLSID\{A65B8071-3BFE-4213-9A5B-491DA4461CA7}\ProgID","",0x00000000,"DxDiag.DxDiagProvider.1"
 HKCR,"CLSID\{A65B8071-3BFE-4213-9A5B-491DA4461CA7}\VersionIndependentProgID","",0x00000000,"DxDiag.DxDiagProvider"
 
+; For "Internet Browser" icon
```

Comment:
If this is for fixing the icon in the livecd, could you instead:
- keep the original code in iexplore.inf (so that when ReatOS gets improved in the future, we can correctly support per-component installation);
- move your additions in hivecls.inf into [livecd.inf](https://git.reactos.org/?p=reactos.git;a=blob;f=boot/bootdata/livecd.inf;h=12fa29cddb74a5cd52eee072ccf12fc06bae0fd2;hb=HEAD) instead? That latter file allows isolating any kind of registry "hacks" or settings specific for the LiveCD.